app/views/buildings/index.html.erb

```diff
@@ -173,7 +175,7 @@
                   <%= link_to "Edit tenant", edit_tenant_path(unit.user), class:"btn btn-primary"%>
                 <% else %>
                   <h5 class="card-title">No tenant</h5>
-                  <%= link_to "Add a tenant", new_tenant_path %>
+                  <%= link_to "Add a tenant", new_tenant_path(unit.user), class:"btn btn-primary" %>
```

Comment:
Can you explain why adding (unit.user), please?

Reply:
i copied the path from the add tenant button on the building index

Reply:
oh okay, the path for "add" is just : `new_tenant_path`
But for "edit", it's : `edit_tenant_path(unit.user)`

Here, the line 178 should be:
`<%= link_to "Add a tenant", new_tenant_path, class:"btn btn-primary" %>`